tools/sizes/code_functions.csv

```diff
@@ -1283,7 +1283,7 @@ asm/non_matchings/code/z_kankyo/func_800FDC94.s,func_800FDC94,0x800FDC94,0x1BF
 asm/non_matchings/code/z_kankyo/func_800FE390.s,func_800FE390,0x800FE390,0x14
 asm/non_matchings/code/z_kankyo/func_800FE3E0.s,func_800FE3E0,0x800FE3E0,0x29
 asm/non_matchings/code/z_kankyo/func_800FE484.s,func_800FE484,0x800FE484,0x5
-asm/non_matchings/code/z_kankyo/func_800FE498.s,func_800FE498,0x800FE498,0x4
+asm/non_matchings/code/z_kankyo/func_800FE498.s,,0x800FE498,0x4
```

Comment:
Hmm, this seems weird. Was this deliberate?

Reply:
nope